### ISSUE TITLE
Suggesting new child term of "Aplasia/Hypoplasia of the sacrum"

### ISSUE THREAD

**Issue by theoriste** [obophenotype/human-phenotype-ontology#3684]
Suggest creating new child term of Aplasia/Hypoplasia of the sacrum [HP:0008517] 
New term label: sacral dysgenesis
New term definition:  partial sacral agenesis, absence of some of the sacral and coccygeal bones
New term comment (if any):
New term synonyms(if any):
New term lay person synonyms (if any):
Reference (e.g., PubMed ID): https://books.google.com/books?id=oG5Fn4Ucus4C&pg=PA207&lpg=PA207&dq=sacral+dysgenesis&source=bl&ots=HcfzGGoHZD&sig=3X37ay2Em0yXpQW2qTPGXiumnpw&hl=en&sa=X&ved=0ahUKEwjLu_u6x8rbAhUE2lMKHZS1BFsQ6AEIggEwDw#v=onepage&q=sacral%20dysgenesis&f=false
Your biocurator ID for nanoattribution (if desired):

**Comment by pnrobinson** [comment]
I think that Sacral dysgenesis is a synonym of Dysplastic sacrum
https://hpo.jax.org/app/browse/term/HP:0008455
I have added sacral dysgenesis as synonyms
and added the definition:
A developmental defect of the sacrum characterized by partial or disordered development of the sacrum in which portions of the sacrum, which normally is formed by fusion of five sacral vertebrae S1-S5, fail to form or fail to form normally.
